### PR TITLE
BUGFIX: ONE-3954 Use explicitly localIdentifiers

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Excplicit localIdentifiers for measures and attributes to avoid re-executions (auto generated localIdentifiers cause diff between old and new AFM)

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs
- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2415
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2205

# Related Jira tasks
- ONE-3954: https://jira.intgdc.com/browse/ONE-3954

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/delivery-promote-packages/job/gooddata-react-components-opensource-repository-check/5898/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 6s

**Comment by ivan-nejezchleb** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/delivery-promote-packages/job/gooddata-react-components-opensource-repository-check/5901/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 6s

File: examples/src/components/LineChartExample.jsx [hunk 1]
```diff
@@ -28,13 +28,21 @@ export class LineChartExample extends Component {
 
     render() {
         const measures = [
-            Model.measure(franchiseFeesIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesAdRoyaltyIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesInitialFranchiseFeeIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesIdentifierOngoingRoyalty).format("#,##0"),
+            Model.measure(franchiseFeesIdentifier)
+                .format("#,##0")
+                .localIdentifier("m0"),
```
Comment (no23reason) [inline examples/src/components/LineChartExample.jsx]: Can you please use something like `"franchiseFeesIdentifier"` like in other files?

File: examples/src/components/PivotTableExample.jsx [hunk 2]
```diff
@@ -39,23 +39,34 @@ export class PivotTableExample extends Component {
...
+                      .localIdentifier("franchiseFeesOngoingRoyalty"),
               ]
             : [];
 
         const attributes = withAttributes
             ? [
-                  Model.attribute(locationStateDisplayFormIdentifier),
-                  Model.attribute(locationNameDisplayFormIdentifier),
-                  Model.attribute(menuCategoryAttributeDFIdentifier),
+                  Model.attribute(locationStateDisplayFormIdentifier).localIdentifier("locationStat"),
```
Comment (no23reason) [inline examples/src/components/PivotTableExample.jsx]: Typo: locationStat**e**

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/delivery-promote-packages/job/gooddata-react-components-opensource-repository-check/5903/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 4s

**Comment by ivan-nejezchleb** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4611/ : SUCCESS in 14m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5899/ : SUCCESS in 14s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1177/ : FAILURE in 39s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2473/ : SUCCESS in 15m 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4625/ : SUCCESS in 9m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1178/ : SUCCESS in 25s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1500/ : SUCCESS in 17m 51s

**Pull request merged**